src/ontology/uberon-edit.obo

```diff
@@ -165877,9 +165872,9 @@ xref: SCTID:280206003
 xref: Wikipedia:Inferior_branch_of_oculomotor_nerve
 intersection_of: UBERON:0001021 ! nerve
 intersection_of: branching_part_of UBERON:0001643 ! oculomotor nerve
-intersection_of: supplies UBERON:0006322 ! inferior rectus extraocular muscle
-relationship: supplies UBERON:0001602 {source="AAO", source="Wikipedia"} ! medial rectus extraocular muscle
-relationship: supplies UBERON:0006322 {source="AAO", source="Wikipedia"} ! inferior rectus extraocular muscle
+intersection_of: innervates UBERON:0006322 ! inferior rectus extraocular muscle
+relationship: innervates UBERON:0001602 {source="AAO", source="Wikipedia"} ! medial rectus extraocular muscle
+relationship: innervates UBERON:0006322 {source="AAO", source="Wikipedia"} ! inferior rectus extraocular muscle
```

Comment:
This seems redundant based on the equivalence axiom.

Reply:
Yes. Happy to remove that, this would not change anything.